docs/document/content/user-manual/shardingsphere-jdbc/usage/sharding/hint.cn.md

```diff
@@ -57,9 +57,6 @@ HintManager hintManager = HintManager.getInstance();
 - 使用 `hintManager.addDatabaseShardingValue` 来添加数据源分片键值。
 - 使用 `hintManager.addTableShardingValue` 来添加表分片键值。
 
-> 分库不分表情况下，强制路由至某一个分库时，可使用 `hintManager.setDatabaseShardingValue` 方式添加分片。
```

Comment:
The first line should not be removed, `hintManager.setDatabaseShardingValue` is working now.

Reply:
I see. I will keep the first line.